analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -240,7 +244,35 @@ open class Npm(
                     .url("https://registry.npmjs.org/$encodedName")
                     .build()
 
-                OkHttpClientHelper.execute(HTTP_CACHE_PATH, pkgRequest).use { response ->
+                OkHttpClientHelper.execute(HTTP_CACHE_PATH, pkgRequest) {
+                    val npmrcFile = getUserHomeDirectory().resolve(".npmrc")
+                    if (npmrcFile.isFile) {
+                        val httpRegex = Regex("^https?://.+$")
```

Comment:
This could go as a constant to the companion object to avoid recompiling the regex for each request.

Reply:
Done.